pkg/relayer/session/interface.go

```diff
@@ -0,0 +1,49 @@
+package session
+
+import (
+	"github.com/pokt-network/smt"
+
+	"pocket/pkg/observable"
+	sessiontypes "pocket/x/session/types"
+)
+
+// RelayerSessions is an interface for managing the relayer's sessions and Sparse Merkle Sum Trees (SMSTs).
+// It provides notifications about closing sessions that are ready to be claimed, and handles the creation
+// and retrieval of SMSTs for a given session.
+type RelayerSessions interface {
+	// ClosingSessions returns an observable that notifies about sessions ready to be claimed.
+	ClosingSessions() observable.Observable[SessionTree]
+
```

Comment:
```suggestion

// TODO: Add `ExpiringSessions` to notify of sessions whose proof submission window is closing.

```

Reply:
Adding it to the interface. Was planning to handle it a bit differently but I think this is a better design.

Please confirm (form its comment) that this is what you intended it to be.